flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/nodes/physical/batch/BatchExecExchange.scala

```diff
@@ -116,25 +75,39 @@ class BatchExecExchange(
 
   override def explainTerms(pw: RelWriter): RelWriter = {
     super.explainTerms(pw)
-      .itemIf("exchange_mode", requiredExchangeMode.orNull,
-        requiredExchangeMode.contains(DataExchangeMode.BATCH))
+      .itemIf("shuffle_mode", requiredShuffleMode.orNull,
+        requiredShuffleMode.contains(ShuffleMode.BATCH))
   }
 
   //~ ExecNode methods -----------------------------------------------------------
 
-  def setRequiredDataExchangeMode(exchangeMode: DataExchangeMode): Unit = {
-    require(exchangeMode != null)
-    requiredExchangeMode = Some(exchangeMode)
+  def setRequiredShuffleMode(shuffleMode: ShuffleMode): Unit = {
+    require(shuffleMode != null)
+    requiredShuffleMode = Some(shuffleMode)
   }
 
-  private[flink] def getDataExchangeMode(tableConf: Configuration): DataExchangeMode = {
-    requiredExchangeMode match {
-      case Some(mode) if mode eq DataExchangeMode.BATCH => mode
-      case _ => DataExchangeMode.PIPELINED
+  private[flink] def getShuffleMode(tableConf: Configuration): ShuffleMode = {
+    requiredShuffleMode match {
+      case Some(mode) if mode eq ShuffleMode.BATCH => mode
+      case _ => getShuffleModeFromConfig(tableConf)
+    }
+  }
+
+  private def getShuffleModeFromConfig(tableConf: Configuration): ShuffleMode = {
```

Comment:
This method name is almost the same with previous one, it's confusing

Reply:
fixed